ansible/roles/graph-protocol/files/values-stg.yaml

```diff
@@ -0,0 +1,11 @@
+# ethereum network
+network: sokol
```

Comment:
This change for chiado, once we setup everyting

Reply:
yup, maybe for now it's better to delete these files...

Reply:
Ok, we'll create an issue